test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -0,0 +1,335 @@
+						sleepFor(10 * SECOND),
+						withOpContext( (spec, log) -> {
+							log.info("Now get all {} accounts created and save it in spec", totalAccounts);
+							for(int i = totalAccounts - 1; i >=0; i-- ) {
+								var op = getAccountBalance(ACCT_NAME_PREFIX + i)
+										.hasAnswerOnlyPrecheckFrom(permissiblePrechecks)
+										.persists(true)
+										.noLogging();
+
+								allRunFor(spec, op);
+							}
+						}),
+						sleepFor(10 * SECOND),
+						exportAccountBalances(() -> ACCOUNT_FILE_EXPORT_DIR ),
+
+						freeze().payingWith(GENESIS)
+								.startingIn(10).seconds()
+								.andLasting(2).minutes()
+
+						);
+	}
+
+	private Function<HapiApiSpec, OpProvider> accountsCreate(PerfTestLoadSettings settings) {
+		totalTestTokens = settings.getTotalTokens() > 10 ? settings.getTotalTokens() : TOTAL_TEST_TOKENS;
+		totalTreasureAccounts = totalTestTokens;
+		totalAccounts = settings.getTotalAccounts() > 100 ? settings.getTotalAccounts() : TOTAL_ACCOUNT;
+
+		log.info("Total accounts: {}", totalAccounts);
+		log.info("Total tokens: {}", totalTestTokens);
+
+		AtomicInteger createdSofar = new AtomicInteger(0);
+
+		return spec -> new OpProvider() {
+			@Override
+			public List<HapiSpecOperation> suggestedInitializers() {
+				log.info("Now running accountsCreate initializer");
+				return Collections.emptyList();
+			}
+
+			@Override
+			public Optional<HapiSpecOperation> get() {
+				int next;
+				if ((next = createdSofar.getAndIncrement()) >= totalAccounts) {
+					return Optional.empty();
+				}
+
+				var op =  cryptoCreate(String.format("%s%s",ACCT_NAME_PREFIX , next))
+						.balance((long)(r.nextInt((int)ONE_HBAR) * 1000 + MIN_ACCOUNT_BALANCE))
+						.key(GENESIS)
+						.fee(A_HUNDRED_HBARS)
+						.withRecharging()
+						.rechargeWindow(30)
+						.noLogging()
+						;
+				if (next > 0) {
+					op.deferStatusResolution();
+				}
+				return Optional.of(op);
+			}
+		};
+	}
+
+	private Function<HapiApiSpec, OpProvider> tokensCreate(PerfTestLoadSettings settings) {
+		AtomicInteger createdSofar = new AtomicInteger(0);
+
+		return spec -> new OpProvider() {
+			@Override
+			public List<HapiSpecOperation> suggestedInitializers() {
+				log.info("Now running tokensCreate initializer");
+				return Collections.emptyList();
+			}
+
+			@Override
+			public Optional<HapiSpecOperation> get() {
+				int next;
+				if ((next = createdSofar.getAndIncrement()) >= totalTestTokens) {
+					return Optional.empty();
+				}
+				var payingTreasury = String.format(ACCT_NAME_PREFIX + next);
+				var op = tokenCreate(TOKEN_NAME_PREFIX + next)
+						.signedBy(GENESIS)
+						.fee(A_HUNDRED_HBARS)
+						.initialSupply(MIN_TOKEN_SUPPLY + r.nextInt(MAX_TOKEN_SUPPLY))
+						.treasury(payingTreasury)
+						.hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS)
+						.hasPrecheckFrom(DUPLICATE_TRANSACTION, OK)
+						.hasKnownStatusFrom(SUCCESS, TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT,
+								TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED, FAIL_INVALID)
```

Comment:
`FAIL_INVALID` is supposed to reserved for unhandled exceptions on the server side; we want clients to fail *loudly* if they ever get that status code. :/

Reply:
I see. removed this status code.